src/main/java/it/pagopa/pm/gateway/service/CcResumeStep1Service.java

```diff
@@ -61,6 +63,9 @@ public class CcResumeStep1Service {
     @Autowired
     private ObjectMapper objectMapper;
 
+    @Autowired
+    private ClientsConfig clientsConfig;
+
```

Comment:
Prefer constructor injection to field injection

Reply:
d90fcfd449c0e4e67ebb47fc70754c86b8bb30f4